x-pack/legacy/plugins/ml/public/application/datavisualizer/index_based/components/field_data_card/_field_data_card.scss

```diff
@@ -1,52 +1,94 @@
 .mlFieldDataCard {
```

Comment:
A lot of these styles are shared by the cards on the file based data viz, so some edits are needed here:

![image](https://user-images.githubusercontent.com/7405507/72257105-b56ee680-3602-11ea-8596-87295a60fd73.png)

Reply:
Fixed in https://github.com/elastic/kibana/pull/54358/commits/35f4033c1f5f9c2416a7e62c7bb2c40b0ba067e2

Reply:
The styling of the progress bars must have been lost, as they aren't showing up:

![image](https://user-images.githubusercontent.com/7405507/72343196-317e3280-36c6-11ea-9551-16dd3e191e24.png)

Also the cards are about 30px wider on the file data viz compared to the previous version (about 380px compared to 350px). Possibly extra padding?

Reply:
https://github.com/elastic/kibana/pull/54358/commits/34ddc1f1b3ed76c4f0003e397730bdc9a0b03444